src/views/AddExpenses.js

```diff
@@ -102,7 +102,7 @@ const AddExpenses = ({ addExpense, setNewExpenseMonth }) => {
                 type="number"
                 id="amount"
                 name="amount"
-                placeholder="... 19,99 ..."
+                placeholder="e.g. 19.99"
```

Comment:
Do we want to change `Expense amount in €` to `Expense amount` here? the `span` is already showing `€` before the input

Reply:
Both fine for me...

Reply:
please remove the € from the placeholder

Reply:
@motine Just to clarify: should I change "Expense amount in €" to "Expense amount" or should I remove the large € icon from the input itself?

Reply:
Done!